Story/AriapetQuests.cs

```diff
@@ -0,0 +1,43 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class AriapetQuests {
+  public IScriptInterface Bot =>IScriptInterface.Instance;
+  public CoreBots Core =>CoreBots.Instance;
+  public CoreStory Story = new();
+
+  public void ScriptMain(IScriptInterface bot) {
+    Core.SetOptions();
+    Storyline();
+    Core.SetOptions(false);
+  }
+
+  public void Storyline() {
+    if (Core.isCompletedBefore(46)) return;
```

Comment:
if (Core.isCompletedBefore(46))
    return;
arrange it like this for better readability